central/processindicator/store/postgres/gen.go

```diff
@@ -1,3 +0,0 @@
-package postgres
-
-//go:generate pg-table-bindings-wrapper --type=storage.ProcessIndicator --search-category PROCESS_INDICATORS --references storage.Deployment
```

Comment:
I'm a bit cautious about doing this, after we've learned that removing the header will make those files eligible for style checking, which they not always pass. In case of process indicator schema it looks fine for now, but maybe it will not be like this in the future. What do you think, how important is it to update the header, having in mind the clear statement about generation status later on and those practical considerations I've mentioned above?

Reply:
I think my preference would be to update this stuff on the process indicators now as painful as it may be.  My reasoning is that we have no idea what else may need to change in process indicators outside of your PLOP work over the next few sprints and we would want it to be as obvious as possible to other contributors as to what is occurring.  That is just one person's opinion of course.